bokun.yaml

```diff
@@ -6454,44 +6852,66 @@ definitions:
       - OTHER
   PricingCategoryDto:
     properties:
-      ticketCategory:
-        $ref: '#/definitions/TicketCategoryEnum'
-      ageQualified:
-        type: boolean
-      defaultCategory:
-        type: boolean
-      dependent:
-        type: boolean
-      flags:
-        type: array
-        items:
-          type: string
-      fullTitle:
-        type: string
       id:
         type: integer
         format: int64
-      internalUseOnly:
+        description: 'Specifies ID of the price category.'
+      title:
+        type: string
+        description: 'Specifies title of the price category.'
+      ticketCategory:
+        $ref: '#/definitions/TicketCategoryEnum'
+      occupancy:
+        type: integer
+        format: int32
+        description: 'Specifies how many places for activity availabilities occupies 1 ticket for this price category.'
+      groupSize:
+        type: integer
+        format: int32
+        description: 'Deprecated.'
```

Comment:
why is this deprecated?

Reply:
In code there is a comment "size of group in case it is a group booking", but when creating price category for group bookings, the field "**occupancy**" reflects group size.
Also, during testing could not make its value different than 0.

Reply:
Tadas helped to make it working.